### PR TITLE
Fix 3.8 compatibility

### LINKED ISSUE #25: Comptability with Kirby 3.8
This plugin isn't compatible with Kirby 3.8 because of the use of `Kirby\Http\Server`, which is deprecated.

See [changelog](https://getkirby.com/releases/3.8#changes-deprecated)

### PR DESCRIPTION
**Pull request by afbora**
Fix #25

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -9,11 +9,11 @@
...
 use microman\Form;
 use microman\FormBlueprint;
-use Kirby\Http\Server;
+use Kirby\Cms\App as Kirby;
 
-\Kirby\Cms\App::plugin('microman/formblock', [
+Kirby::plugin('microman/formblock', [
     'options' => [
-        'from_email' => 'no-reply@' . Server::host(),
+        'from_email' => 'no-reply@' . Kirby::instance()->environment()->host(),
```
Comment (lukasbestle) [inline index.php:16]: @afbora Have you tested that this does not break Kirby's loading order? Calling `App::instance()` from a plugin `index.php` outside of a callback is risky.

Comment (afbora) [reply index.php:16]: Also I've tested this changes, seems works for me both: `Kirby::instance()->environment()->host()` and `(new Environment())->host()`

Comment (lukasbestle) [reply index.php:16]: You are right, should be fine because the environment is always loaded before plugins.

**Review by lukasbestle — APPROVED** [review]
The changes look good to me.

**Comment by youngcut** [comment]
Oh, the kirby core team fixed a plugin that i started. What an honor. 🤗
Thanks. 🙌
May you could take a look to [this issue](https://github.com/youngcut/kirby-form-blocks/issues/23). It's closed but somebody is still [struggling with that](https://forum.getkirby.com/t/display-form-on-website/26590/2).
Maybe, this could be a solution: `<?php /** @var \Kirby\Cms\Block $block */ ?>`

**Comment by afbora** [comment]
You're welcome. I've fixed that issue with this PR, here related line: https://github.com/youngcut/kirby-form-blocks/pull/26/files#diff-c380923c92306a8e0cd1507c552739451f8ac7bca3cb3882f64e7700da11a2ccR600

**Pull request merged**